src/sinks/azure_common/service.rs

```diff
@@ -58,24 +49,14 @@ impl Service<AzureBlobRequest> for AzureBlobService {
 
```

Comment:
In the `poll_ready()` and `call()` function bodies , perhaps add the comment:

```
// Emission of Error internal event is handled upstream by the caller
```

, for consistency. (see https://github.com/vectordotdev/vector/pull/14425/commits/90ec41059d22cd6e78c70620a462b675a9eff5df )